my_game.py

```diff
@@ -656,7 +659,7 @@ def on_update(self, delta_time):
         if self.player_sprite.lives <= 0:
             arcade.stop_sound(self.sound_thrust_player)
             arcade.unschedule(self.spawn_ufo)
-            game_over_view = GameOverView(player_score=self.player_score, level=self.level)
+            game_over_view = GameOverView(player_score=self.player_score, level=level)
```

Comment:
If level is global, why do we need to parse it here? I'd prefer it if it was not global.